fdbserver/IKeyValueStore.h

```diff
@@ -51,12 +51,18 @@ class IKeyValueStore : public IClosable {
 	// The total size of the returned value (less the last entry) will be less than byteLimit
 	virtual Future<Standalone<VectorRef<KeyValueRef>>> readRange( KeyRangeRef keys, int rowLimit = 1<<30, int byteLimit = 1<<30 ) = 0;
 
-	//Returns the amount of free and total space for this store, in bytes
+	// Returns the amount of free and total space for this store, in bytes
+	virtual std::tuple<size_t, size_t, size_t> getSize() { return std::make_tuple(0, 0, 0); }
```

Comment:
```suggestion
	virtual std::tuple<size_t, size_t, size_t> getSize() = 0;
```
The comments of this function seems to be swapped with `getStorageBytes`.